cmd/models/get.go

```diff
@@ -80,4 +82,10 @@ var getCmd = &cobra.Command{
 
 func init() {
 	getCmd.Flags().String("model-id", "", "Authorization Model ID")
+	getCmd.Flags().String("store-id", "", "Store ID")
+
+	if err := getCmd.MarkFlagRequired("store-id"); err != nil {
+		fmt.Println(fmt.Errorf("error setting flag as required - %v: %w", "cmd/models/get", err))
```

Comment:
why do we need `fmt.Errorf` if we are go have fmt.Println anyways?
Same comment for other files as well.

Reply:
Will change to Sprintf

Reply:
Changed